mmdet/datasets/coco.py

```diff
@@ -542,3 +542,34 @@ def evaluate(self,
         if tmp_dir is not None:
             tmp_dir.cleanup()
         return eval_results
+
+    def __repr__(self):
+        """Print the number of instance number."""
+        dataset_type = 'Test' if self.test_mode else 'Train'
+        result = f'\n{dataset_type} dataset ' \
```

Comment:
There may be two issues here. 
1. brackets are preferred rather than backslash for newlines. 
2. Better add a class name in the docstring as well. 

```python
result = (f'\n{dataset_type} dataset {self.__class__.__name__}' 
    f'with number of images {len(self)}, '
    f'and instance counts: \n')
```

Reply:
will fix this soon